src/components/Element.js

```diff
@@ -82,7 +82,7 @@ const Element = (type: string, hocOptions: {sourceType?: string} = {}) =>
       this._element.destroy();
       this.context.unregisterElement(this._element);
     }
-    props: Props;
+
```

Comment:
Maybe I'm missing something, but what did this get removed for? Seems like maybe we're not being as strict with types as we could be, but maybe we're making up for this somewhere else.

Reply:
New Flow does not require Props to be declared this way. (See the class signature)